code/modules/mob/living/carbon/human/species_types/angel.dm

```diff
@@ -136,4 +136,96 @@
 		H.movement_type &= ~FLYING
 		override_float = FALSE
 		H.pass_flags &= ~PASSTABLE
-		H.CloseWings()
\ No newline at end of file
+		H.CloseWings()
+
+
+
+/datum/species/dullahan
+	name = "dullahan"
+	id = "dullahan"
+	default_color = "FFFFFF"
+	species_traits = list(EYECOLOR,HAIR,FACEHAIR,LIPS)
+	mutant_bodyparts = list("tail_human", "ears", "wings")
+	default_features = list("mcolor" = "FFF", "tail_human" = "None", "ears" = "None", "wings" = "None")
+	use_skintones = 1
+	no_equip = list(slot_back)
+	mutant_brain = /obj/item/organ/brain/dullahan
+	mutanteyes = /obj/item/organ/eyes/dullahan
+	mutanttongue = /obj/item/organ/tongue/dullahan
+	mutantears = /obj/item/organ/ears/dullahan
+	blacklisted = 1
+	limbs_id = "human"
+	skinned_type = /obj/item/stack/sheet/animalhide/human
+
+	var/obj/item/bodypart/head/myhead
+
+/datum/species/dullahan/on_species_gain(mob/living/carbon/human/H, datum/species/old_species)
+	..()
+	var/obj/item/bodypart/head/head = H.get_bodypart("head")
+	if(head)
+		myhead = head
+		head.drop_limb()
+		var/obj/item/dullahan_relay/DR = new (myhead)
+		DR.owner = H
+
+/datum/species/dullahan/spec_life(mob/living/carbon/human/H)
+	if(myhead)
+		var/obj/item/organ/eyes/eyes = H.getorganslot(ORGAN_SLOT_EYES)
+		if(eyes.tint)
+			H.reset_perspective(H)
+		else
+			H.reset_perspective(myhead)
+
+	//	if(myhead in view(7, get_turf(H)))
+//			H.disabilities &= ~DEAF
+//		else
+//			H.disabilities |= DEAF
+	else
+		H.gib()
+
+
+/obj/item/organ/brain/dullahan
+	decoy_override = TRUE
+	vital = FALSE
+
+/obj/item/organ/tongue/dullahan
+	zone = "abstract"
+
+/obj/item/organ/tongue/dullahan/TongueSpeech(var/message)
+	if(ishuman(owner))
+		var/mob/living/carbon/human/H = owner
+		if(H.dna.species.id == "dullahan")
+			var/datum/species/dullahan/D = H.dna.species
+			D.myhead.say(message)
+	message = ""
+	return message
+
+
+/obj/item/organ/ears/dullahan
+	zone = "abstract"
+
+/obj/item/organ/eyes/dullahan
+	name = "head vision"
+	desc = "An abstraction."
+	actions_types = list(/datum/action/item_action/organ_action/use)
+	zone = "abstract"
+
+/obj/item/organ/eyes/night_vision/ui_action_click()
+	if(tint)
+		tint = 0
+	else
+		tint = INFINITY
+	owner.update_sight()
+
+
+/obj/item/dullahan_relay
+	var/mob/living/owner
+
+/obj/item/dullahan_relay/Hear(message, atom/movable/speaker, message_langs, raw_message, radio_freq, list/spans)

```

Comment:
I put a bunch of debug logs in this, it doesn't ever heard the message, even if you place it on the ground outside a head

Reply:
Oh I need the hear flag nevermind